_includes/icon-github.html

```diff
@@ -0,0 +1 @@
+<a href="https://github.com/{{ include.username }}"><span class="icon icon--github">{% include icon-github.svg %}</span><span class="username">{{ include.username }}</span></a>
```

Comment:
I'd prefer this 
![screen shot 2016-07-07 at 19 18 08](https://cloud.githubusercontent.com/assets/216452/16662365/99e3172c-4477-11e6-8170-fb9c91122bd0.png)

to this 
![screen shot 2016-07-07 at 19 18 20](https://cloud.githubusercontent.com/assets/216452/16662369/9db52da4-4477-11e6-8643-8822af0c2caf.png)